README_CONTAINER_IMAGE.md

```diff
@@ -38,4 +44,20 @@ Here is an example of how to run a containerized `openshift-ansible` playbook th
            -e PLAYBOOK_FILE=playbooks/certificate_expiry/default.yaml \
            openshift/openshift-ansible
 
-The [playbook2image examples](https://github.com/aweiteka/playbook2image/tree/master/examples) provide additional information on how to use an image built from it like this one.
+#### Scheduled job
+
+The certificate expiration checks in the [standalone container example](#standalone-container) above can also run automatically from within OpenShift on a regular basis using the [example](playbooks/certificate_expiry/scheduled-certcheck.yaml) [CronJob](https://docs.openshift.org/latest/dev_guide/cron_jobs.html) available in the certificate expiration playbooks directory, which will run a playbook that generates certificate expiration reports and upload them to the masters.
```

Comment:
Yes, thanks for noting.
Longer story: as upstream is already at the CronJob level I started this with CronJob - later tests showed that the practical approach at this point would be ScheduledJob and I updated the object but forgot to update the doc.